package.json

```diff
@@ -46,13 +40,20 @@
     "@eslint/js": "~9.23.0",
     "@reporters/github": "^1.7.2",
     "@testing-library/react": "~16.3.0",
-    "@types/node": "22.14.0",
-    "@types/react": "^19.1.0",
-    "@types/react-dom": "^19.1.1",
+    "cross-env": "^7.0.3",
     "eslint": "~9.23.0",
     "eslint-plugin-import-x": "~4.10.0",
-    "eslint-plugin-no-relative-import-paths": "~1.6.1",
     "prettier": "3.5.3",
     "prettier-plugin-tailwindcss": "0.6.11"
+  },
+  "packageManager": "pnpm@10.10.0",
+  "engines": {
+    "node": "v22"
+  },
+  "devEngines": {
+    "packageManager": {
+      "name": "pnpm",
+      "version": "10.8.0+sha256.29bf2c5ceaea7991ee82eec15fe7162e0fad816d0c4a6b35a16c01d39274bf69"
```

Comment:
We don't have the sha in `packageManager` but we do here? I'd vote for not having the sha in either place?

Reply:
pnpm automatically sets the other field when updating, but not this one. I'll update this one accordingly